build.gradle

```diff
@@ -40,16 +40,16 @@ repositories {
 
 dependencies {
     compile "io.appium:java-client:5.0.4"
-    compile "org.jbehave:jbehave-core:4.1.1"
-    compile "org.jbehave:jbehave-spring:4.1.1"
+    compile "org.jbehave:jbehave-core:4.3.3"
+    compile "org.jbehave:jbehave-spring:4.3.3"
     compile "org.projectlombok:lombok:1.16.18"
     compile "com.opencsv:opencsv:3.9"
     compile "ru.yandex.qatools.allure:allure-junit-adaptor:1.5.4"
     compile "org.slf4j:slf4j-api:1.7.25"
     testCompile "org.springframework:spring-test:4.3.8.RELEASE"
     testCompile "junit:junit:4.12"
     testCompile "org.mockito:mockito-core:2.+"
-    testCompile "de.codecentric:jbehave-junit-runner:1.2.0"
+    testCompile "com.github.valfirst:jbehave-junit-runner:2.1.4"
```

Comment:
We have lost sight of this dependence. It is not used anywhere, delete it please